src/asimo_control/script/camera.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+
+#This code is modified from : #https://gist.github.com/rethink-imcmahon/77a1a4d5506258f3dc1f
+
+# rospy for the subscriber
+import rospy
+# ROS Image message
+from sensor_msgs.msg import Image
+# ROS Image message -> OpenCV2 image converter
+from cv_bridge import CvBridge, CvBridgeError
+# OpenCV2 for saving an image
+import cv2
+import time
+
+# Instantiate CvBridge
+bridge = CvBridge()
+
```

Comment:
why camera stuff is here ?

Reply:
Hello Mike, 
For this brach, I have removed that files you mentioned. 
For the testing task, please check ' rostest ' folder.